pages/code-review.md

```diff
@@ -1,6 +1,143 @@
 ---
 permalink: /code-review/
-title: Code Review
+title: Code Review Guide
 ---
 
-# Code Review
+> A friendly guide for reviewing code--and not each other--at 18f.
+
+(Forked from the excellent Consumer Financial Protection Bureau
+[guide](https://github.com/cfpb/front-end/blob/master/code-reviews.md)). This
+is a **WORK IN PROGRESS**. Help us make it better in the
+[#wg-code-reviews](https://18f.slack.com/messages/wg-code-reviews/) channel!
+
+
+## Why reviews?
+
+Code reviews are incredibly important step to the development process. Our goals
+when reviewing code are to:
+
+- expose bugs before they make it to production;
+- ensure a consistent quality in the codebase;
+- create an environment for sharing knowledge and developing skills;
+- encourage open communication between the entire team.
+
+
+## Tips for a successful review
+
+For a code review to be successful, we all need to be on the same page. To help
+accomplish this, everyone should follow these tips.
+
+### For everyone
+
+- Remember that we're reviewing the code, not the author. Nothing should be
+  given or taken taken personally.
+- There is often more than one way to approach a solution. Discuss tradeoffs and
+  reach a resolution quickly.
+- Refer to the style guides whenever there's a discussion that can't be resolved.
```

Comment:
given that we don't have many of these just yet....maybe we should remove this for now but come back to it